README.md

```diff
@@ -33,6 +33,11 @@ return (
     />
   </div>
 );
+
+// Prefer multiple useData calls above, but when the data you need to fetch is dynamic in its length, useMultipleData can help:
+const [data, update] = useMultipleData({ tasks: ["1", "3"], jobs: ["2"] });
```

Comment:
```suggestion
const [data, update] = useMultipleData({ tasks: ["1", "3"], jobs: ["2"] });
// where `tasks` and `jobs` are atom kinds and strings in the array values are atom ids
```

Or something like that. I think we should explain the interface a little to the use in the readme.

Reply:
I mean, I thought the interface was obvious, but there's two separate things going on here:
1) Multiple data kinds (`tasks` and `jobs` here).
2) Multiple ids in those kinds (the arrays).

I don't understand why these are grouped. IOW, the syncing ramifications are non-obvious to me. Are these all being batched? But doesn't `useData` already batch?

For multiple IDs, it seems to me that this would be better:
```javascript
useData("tasks", ["1", "3"])
useData("jobs", ["2"])
useData("other", "1")
```
where `useData` takes a scalar or an array for the IDs. And then they'd be accessed like so:
```
const [data] = useData("tasks", ["1", "3"])
const [task1, task3] = data;
```

Reply:
Mostly, I think it'd be better to be reusing or reworking the existing `useData`. But I'd like to understand the problem a bit better, first.

Reply:
We have this outstanding task to improve documentation, so you’re right. At the same time, you did understand the design, didn’t you? :-)

Reply:
I think the suggestion about a hook that takes single kind, multiple ids is great. @tjklemz, thanks!

I don’t want to overload the simple `useData`, so I have instead overloaded the `useMultipleData`. This ensures that the simple use case is simple, and the advanced use cases are possible.

I have also slightly reshaped the return type. I think having the return `tasks` array be identified by the index in the array leads to awkward work aligning the id and the array index, so I instead return a map.

Please see the updated docs and test!